tests/docker-compose.yml

```diff
@@ -10,3 +10,5 @@ services:
             - mender-device-adm
             - mender-device-auth
             - mender-inventory
+    mender-device-auth:
```

Comment:
Can you add a note that we're assuming that image `mendersoftware/deviceauth` was built locally and tagged under this name?